### PR TITLE
Fix issue 7

### LINKED ISSUE #7: README.md: Add a step to use a virtual environment.
This would be the ideal way.

### PR DESCRIPTION
**Pull request by ashwani99**
Fixes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,14 @@ First, clone the repository:
...
 
+Create a [virtualenv](https://pypi.python.org/pypi/virtualenv) and activate it to isolate the working environment
+    
+    virtualenv venv
+    source venv/bin/activate
+
 cd to the repository
 
-    cd pclubuiet
+    cd website
```
Comment (divyam3897) [inline README.md]: Remove this change

File: README.md [hunk 2]
```diff
@@ -7,9 +7,14 @@ First, clone the repository:
 
     git clone https://github.com/pclubuiet/website.git
 
+Create a [virtualenv](https://pypi.python.org/pypi/virtualenv) and activate it to isolate the working environment
+
```
Comment (divyam3897) [inline README.md]: Make two seperate steps for it

**Comment by ashwani99** [comment]
@divyam3897 Made the changes as you suggested. Is it okay now?

**Comment by divyam3897** [comment]
Fixed in https://github.com/pclubuiet/website/pull/30 Thank you for your PR

**Pull request closed without merging**